src/main.c

```diff
@@ -10,6 +10,8 @@
 #include <locale.h>
 #include <stdio.h>
 #include <string.h>
+#define SYSLOG_NAMES
```

Comment:
Is this really needed? https://sourceware.org/bugzilla/show_bug.cgi?id=16355 sounds like we should avoid this define.

Reply:
The define opens up the use of `facilitynames[]`.  It has been a long standing practice, but we can replace that with a local array of the same names if you want.